medcat/meta_cat.py

```diff
@@ -421,18 +473,37 @@ def prepare_document(self, doc: Doc, input_ids: List, offset_mapping: List, lowe
             start = ent.start_char
             end = ent.end_char
 
+            flag = 0
```

Comment:
This change in behaviour seems to affect both LSTM and BERT approaches.
Is that deliberate? If so, it would require some documentation since it looks like new/changed behaviour.
If no, then we'd need to make sure this only runs for BERT.

Reply:
Yes, it changes the way we extract the tokens for the medical entity in question which should affect LSTM and BERT.
I've optimised the implementation further and added comments